scripts/teeracle1.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
```

Comment:
Why is this called teeracle**1** instead of plain teeracle?

Reply:
Beats me - this was just moved from the teeracle repo. I can re-name it, if you wish.